packages/components/src/ErrorPage/_docs/ErrorPage.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { ComponentDocsTemplate } from "../../../../../storybook/components/DocsContainer"
+import { ErrorPage } from "../index"
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Components/ErrorPage",
+  component: ErrorPage,
+  parameters: {
+    docs: {
+      container: ComponentDocsTemplate,
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+    installation: [
+      "npm install @kaizen/components",
+      "import { ErrorPage } from `@kaizen/components`",
+    ],
+    resourceLinks: {
+      /** @todo: Add Github link (adjust as needed) */
+      sourceCode:
+        "https://github.com/cultureamp/kaizen-design-system/tree/master/packages/components/src/ErrorPage",
+    },
+  },
+  args: {
+    callToAction: {
+      onContactSupport: () => alert("Custom handler"),
+    },
+  },
+} satisfies Meta<typeof ErrorPage>
+
+export default meta
+
+export const Error400: StoryObj<typeof meta> = {
```

Comment:
Perhaps these should be in a stickersheet story? I had originally removed that file because I didn't have much more to show apart from the main component story. Let me know if you want them moved.

Reply:
I think the best tests for this are the visual snapshot tests, so that's why I've added these stories.

Reply:
This is a bit awkward, we do want consumers to be able to see the outcome for each code, but at the same time this seems expensive.

Let's do a sitckersheet, but for the docs can we simply output the mapping in a table of some sort? Just to represent what to expect when providing the error code